_opensearch/query-dsl/text-analyzers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Text analyzers
+parent: Query DSL
+nav_order: 41
+---
+
+
+# Text analyzers
+
+OpenSearch applies text analysis during indexing or searching for `text` fields. There is a standard  analyzer that OpenSearch uses by default for text analysis. To optimize unstructured text for search, you can convert it into structured text with our specialized text analyzers.
+## Specialized analyzers
```

Comment:
Add a newline before this line

Reply:
I'm not sure the modifier "specialized" is needed here.

Reply:
We are differentiating ourselves from Elastic's "built-in" analyzers. We also have these, they are called "specialized" analyzers.